tests/integration/test_nlp/configs/config.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<yandex>
+	<logger>
+		<level>trace</level>
+		<log>/var/log/clickhouse-server/clickhouse-server.log</log>
+		<errorlog>/var/log/clickhouse-server/clickhouse-server.err.log</errorlog>
+		<size>1000M</size>
+		<count>10</count>
+	</logger>
+
+    <tcp_port>9000</tcp_port>
+    <listen_host>127.0.0.1</listen_host>
+
+    <openSSL>
+        <client>
+            <cacheSessions>true</cacheSessions>
+            <verificationMode>none</verificationMode>
+            <invalidCertificateHandler>
+                <name>AcceptCertificateHandler</name>
+            </invalidCertificateHandler>
+        </client>
+    </openSSL>
+
+    <max_concurrent_queries>500</max_concurrent_queries>
+    <mark_cache_size>5368709120</mark_cache_size>
+    <path>./clickhouse/</path>
+    <users_config>users.xml</users_config>
+
+    <synonyms_extensions>
+        <extension>
+            <name>en</name>
+            <type>plain</type>
+            <path>/etc/clickhouse-server/dictionaries/ext-en.txt</path>
+        </extension>
+        <extension>
+            <name>ru</name>
+            <type>plain</type>
+            <path>/etc/clickhouse-server/dictionaries/ext-ru.txt</path>
+        </extension>
+    </synonyms_extensions>
+
+    <lemmatizers>
+        <lemmatizer>
+            <lang>en</lang>
+            <path>/etc/clickhouse-server/dictionaries/lem-en.bin</path>
```

Comment:
But where is this file?

Reply:
`tests/integration/test_nlp/dictionaries/lem-en.bin`